Automattic/jetpack
# Add Zendesk chat module

Add Zendesk pre-sales chat widget to `my-jetpack`

## Proposed changes:

This adds a conditionally rendered (based on date and time) Zendesk chat widget by embedding the chat widget script

It is configured to only be visible during chat available hours

### Other information:


## Jetpack product discussion

P2: pbNhbs-5cD-p2

## Does this pull request change what data or activity we track or use?

Depends on how that is interpreted, I'm sure the chats are all saved somewhere, so in that sense, yes. No other trackers are being added in addition to that though

## Testing instructions:

1. Checkout this branch
2. Head to `/wp-admin/admin.php?page=my-jetpack`
3. If you are looking at this on a Monday-Friday between 9:00 UTC and 23:00 UTC, ensure the widget shows up in the bottom right corner
 
![image](https://user-images.githubusercontent.com/65001528/218593872-4356e252-d104-4a6b-b317-28fdf307663f.png)

5. You can fudge the numbers in the `utils.ts` script to test when outside of hours if you'd like, but the new automated tests specifically test this, so you could just check if the way the tests are written look good
